workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Dashboard Basics
+---
+
+The Dashboard Builder allows you to generate new dashboards in under a minute, integrate custom widgets into existing dashboards, or effortlessly clone default dashboards for further refinement. Drill into any data ported to Middleware with advanced filtering options by date range and metric value.
+
+# How Dashboards Work
+
+The Dashboard Builder is a Graphical User Interface (GUI) based approach that enables users to create complex query logic without the added stress of learning a new query language. The metrics made available to users when constructing dashboards are dynamically generated based on the data that users port to Middleware. For example, if you have instrumented Middleware's Infrastructure Agent and APM packages, you will have access to all of your host and application-level data. 
```

Comment:
Sub "port" for "ingest". Also suggest making the "For example..." sentence a new line. This is really a mintlify issue as it makes it a sad looking block text as is.